index.html

```diff
@@ -0,0 +1,187 @@
+    <main>
+
+      <!-- #### hero-section #### -->
+      <section class="hero-section">
+        <h2 class="headline-heading1">"Hello! technology geeks"</h2>
+        <h2 class="headline-heading2">
+          BECOME A PROFESSIONAL REMOTE WEB DEVELOPER
+        </h2>
+        <p class="headline-details">
+          Join the community of world class web developer around the world, and
+          become a professional remote web developer. Change your career into
+          tech now
+        </p>
+        <div class="headline-img-container">
+          <div>
+        <h2 class="headline-heading3">NEXT COHORT STARTING IN 15 DAYS</h2>
+        <p class="headline-details2">Book your enrolment now</p>
+        </div>
+        <img class="hero-img" alt="hero image" src="images/hero-bg.png">
+        </div>
+      </section>
+
+      <!-- #### highlight-content-sectionn #### -->
+      <section class="highlight-section" id="program">
+        <h2 class="highlight-heading">Why THECODECHASER</h2>
+        <hr>
+        <div class="hightlight-content">
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon1.png">
+            <h3 class="highlight-item-heading">Projects</h3>
+            <p class="highlight-item-details">We believe in project based learning, you build industry level projects.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon2.png">
+            <h3 class="highlight-item-heading">Material</h3>
+            <p class="highlight-item-details">We have best open-source study material from internet, That will teach you.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon3.png">
+            <h3 class="highlight-item-heading">Patern</h3>
+            <p class="highlight-item-details">Our study pattern is diffrent from many online school, we focus on mastry learning.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon4.png">
+            <h3 class="highlight-item-heading">Jobs</h3>
+            <p class="highlight-item-details">Our career coaches are help you to find your dream job, Get high paying jobs.</p>
+          </div>
+          <div class="highlight-item">
+            <img alt="highlight icon" class="highlight-item-icon" src="images/highlight-icon5.png">
+            <h3 class="highlight-item-heading">Help</h3>
+            <p class="highlight-item-details">Our staff is available for you all the time to help you with quastions.</p>
+          </div>
+        </div>
+        <button type="button" class="button join-us">Enroll now and confirm your spot</button>
+        <a class="highight-end" href="#">SEE THE WHOLE PROGRAM</a>
+      </section>
```

Comment:
- [ ] Kindly update the used tags within this section to use an `<ul;>` along with `<li>`s and `<article>` (for each card) to give a semantic meaning for the section. The usage of `div` tags is a bad practice.
- [ ] Please make sure to horizontally center this `section` to match the design.

Reference Design
![image](https://user-images.githubusercontent.com/33767951/142401578-392b04df-d8c9-4320-8a53-d9e302823bca.png)

Your Styling => `786`
![image](https://user-images.githubusercontent.com/33767951/142401480-1cbda7a9-56e9-498a-b947-d6132e00ba7c.png)